web/pdf_page_view.js

```diff
@@ -612,9 +644,10 @@ class PDFPageView {
     keepAnnotationEditorLayer = false,
     keepXfaLayer = false,
     keepTextLayer = false,
+    extraCancelDelay = 0,
```

Comment:
Nit: Maybe `cancelExtraDelay` instead, since it's the `extraDelay` that we're passing to the `cancel`-method?